tests/flow/test_graph_create.py

```diff
@@ -148,6 +147,13 @@ def test09_create_use_alias_in_many_clauses(self):
             except redis.exceptions.ResponseError as e:
                 self.env.assertContains("The bound variable 'r' can't be redeclared in a CREATE clause", str(e))
 
+        query = "CREATE (n1)-[r:Rel1]->(n2), (n2)-[r:Rel1]->(n1)"
+        try:
+            redis_graph.query(query)
+            self.env.assertTrue(False)
+        except redis.exceptions.ResponseError as e:
+            self.env.assertContains("Variable `r` already declared", str(e))
```

Comment:
why different error message?

Reply:
In this case (a single CREATE clause) the validation first spots the fact that `r` is already defined.
in the other cases multiple CREATE clauses `r` is bounded.